Furball.Engine/Engine/Graphics/Drawables/BaseDrawable.cs

```diff
@@ -41,7 +42,8 @@ public abstract class BaseDrawable {
         /// <summary>
         ///     Radius of the Circle (Used for Click detection and other hitboxes)
         /// </summary>
-        public float CircleRadius = 0f;
+        [CanBeNull]
+        public Bindable<float> CircleRadius = new(0f);
```

Comment:
Why did i put this here? this should *NEVER* be null

Reply:
i have no idea, maybe rider added it automatically or smth? it happened to me once aswell,

either way, removed